src/tribler-core/tribler_core/modules/version_check/versioncheck_manager.py

```diff
@@ -84,6 +82,10 @@ async def check_new_version_api(self, version_check_url):
         except asyncio.TimeoutError:
             self._logger.warning("Checking for new version failed for %s", version_check_url)
         except ValueError as ve:
-            raise ValueError(f"Failed to parse Tribler version response.\nError:{ve}") from ve
+            self._logger.warning(f"Failed to parse Tribler version response.\nError:{ve}")
+        except Exception as e:  # pylint: disable=broad-except
```

Comment:
Probably it is better to catch just `ClientPayloadError` here or delete all previous exception handlers as well and replace them with a single broad Exception handler.

Reply:
Fixed by replacing all previous exceptions with a single broad Exception handler.